dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentTypeLoader.java

```diff
@@ -0,0 +1,303 @@
+    private long fakeInode = Long.MAX_VALUE;
+    private String fakeTitle = "Content Title";
+
+
+
+    public void invalidate(ContentType contentType, PageMode mode) {
+        removeContentTypeFile(contentType);
+    }
+
+
+
+    public InputStream buildVelocity(ContentType type, String filePath, PageMode mode ) {
+
+        // let's write this puppy out to our file
+        StringBuilder sb = new StringBuilder();
+
+        // CONTENTLET CONTROLS BEGIN
+        sb.append("#set( $EDIT_CONTENT_PERMISSION =$EDIT_CONTENT_PERMISSION")
+            .append(fakeInode)
+            .append(" )");
+        sb.append("#set( $CONTENT_INODE ='")
+            .append(fakeInode)
+            .append("' )");
+        sb.append("#set( $IDENTIFIER_INODE ='")
+            .append(fakeIdentifier)
+            .append("' )");
+
+        // set all properties from the contentlet
+        sb.append("#set( $ContentInode ='")
+            .append(fakeInode)
+            .append("' )");
+        sb.append("#set( $ContentIdentifier ='")
+            .append(fakeIdentifier)
+            .append("' )");
+        sb.append("#set( $ContentletTitle =\"")
+            .append(UtilMethods.espaceForVelocity(fakeTitle))
+            .append("\" )");
+
+        // Structure fields
+        List<Field> fields = type.fields();
+
+
+        for (Field field : fields) {
+
+
+            String contField = field.dbColumn();
+            String contFieldValue = null;
+            Object contFieldValueObject = null;
+            if (UtilMethods.isSet(contField)) {
+                try {
+                    contFieldValueObject = field.name();
+                    contFieldValue = contFieldValueObject == null ? "" : contFieldValueObject.toString();
+                } catch (Exception e) {
+                    Logger.error(this.getClass(), "writeContentletToFile: " + e.getMessage());
+                }
+                if (!(field instanceof DateTimeField || field instanceof DateField || field instanceof TimeField)) {
+                    sb.append("#set( $")
+                        .append(field.variable())
+                        .append(" =\"")
+                        .append(UtilMethods.espaceForVelocity(contFieldValue)
+                            .trim())
+                        .append("\" )");
+                }
+            }
+            String fv = field.values() != null ? field.values() : "";
+            if (field instanceof TextField || field instanceof TextAreaField || field instanceof WysiwygField) {
+                sb.append("#set( $")
+                    .append(field.variable())
+                    .append(" =\"[ #fixBreaks($")
+                    .append(field.variable())
+                    .append(") ]\")");
+            } else if (field instanceof ImageField) {
+                // Identifier id = (Identifier) InodeFactory.getChildOfClassByRelationType(content,
+                // Identifier.class, field.getFieldRelationType());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)